src/deluge/memory/general_memory_allocator.cpp

```diff
@@ -22,7 +22,7 @@
 #include <new>
 #include "io/debug/print.h"
 #include "util/functions.h"
-#include <string.h>
+#include <cstring>
```

Comment:
Do we already depend on this somewhere? Just thinking of binary size

Reply:
`cstring` is just the C++ wrapper for `string.h`. You're not supposed to use C standard headers directly in C++ code, so the C++ standard library provides wrappers that handle all the weird stuff that can crop up.

Reply:
It's header-only so there's no overhead